requirements.txt

```diff
@@ -1,9 +1,10 @@
 fastapi>=0.110.1,<0.111.1
 pydantic-settings>=2.2.1,<2.3.0
 uvicorn[standard]>=0.29.0,<0.30.0
-black>=24.3.0,<24.4.0
+black>=24.3.0,<24.4.2
 geojson-pydantic>=1.0.1,<1.0.2
 rio-tiler>=6.6.1,<6.7.0
 rasterio>=1.3.9,<1.3.10
 geopandas>=1.0.0,<1.0.1
 shapely>=2.0.3,<2.0.4
+starlette>=0.37.2,<0.38.0
```

Comment:
Let's not include this in here, we're only using it explicitly because fastapi is based on it, so it better control those versions too